src/utils/list-components.ts

```diff
@@ -12,7 +12,11 @@ export function constructComponentsList(query?: string): {
     items: ListedComponent[];
     normalizedQuery: string | null;
 } {
-    const normalizedQuery = query ? query.toLowerCase() : null;
+    let normalizedQuery = query ? query.toLowerCase() : null;
```

Comment:
Why not empty string as fallback?

```suggestion
    let normalizedQuery = query ? query.toLowerCase() : '';
```

Reply:
```ts
export function constructComponentsList(query = ''): {
```